config.yaml

```diff
@@ -45,6 +45,8 @@ listeners:
       databaseId: YOUR_SPANNER_DATABASE
       # [Optional] - Global else default to TableConfigurations
       configTableName: TableConfigurations
+      # [Optional] endpoint configuration for spanner
+      endpoint: YOUR_ENDPOINT
```

Comment:
Pls update https://github.com/cloudspannerecosystem/cassandra-to-spanner-proxy/blob/main/docs/config_options.md as well